### ISSUE TITLE
Set `cwd` properly

### ISSUE THREAD

**Issue by torik42** [mfbehrens99/linter-yalafi#17]
Currently, it's not possible to use the YaLafi feature `\LTinput{…}` because the path given to `\LTinput` is interpreted relative to `cwd` which is not set.

For multifile latex documents in atom, the package [`atom-latex`](https://atom.io/packages/latex) allows for magic command
`% !TEX root = relative_path/file.tex` at the beginning of a file, which tells `atom-latex` to build the file `relative_path/file.tex` which usually inputs or includes the current file. Thus, for `\LTinput` and `\input` to be consistent, we should set `cwd` to `relative_path` if the magic command is given.

**Comment by mfbehrens99** [comment]
I was thinking about getting the cwd directly from the atom plugin. I'm using [Atom-Latex by ashthespy](https://github.com/ashthespy/Atom-LaTeX) and this plugin is able to detect the root file automatically without the need of any magic commands. In Atom packages can offer and consume services, so I thought it might be good to directly ask these plugins for the root file.

**Comment by torik42** [comment]
I don’t know, how exactly the Atom-Latex package works. According to the [README](https://github.com/ashthespy/Atom-LaTeX#setting-the-latex-root-file) it also allows to use the magic command to specify the root file.

Does the package already provide a service? Then it’s maybe best to use it, whenever the package is installed. I would not force the user to use a specific LaTeX package for `linter-yalafi` to work. If the service is not yet available, we could open an issue for that. Till then, we could merge this solution, which could be a fallback anyway.

**Comment by mfbehrens99** [comment]
>I don’t know, how exactly the Atom-Latex package works. According to the README it also allows to use the magic command to specify the root file.

Just go and read on. If the manual fails it will search for the magic command. If this fails as well, if will look for a `.latexconfig` if this fails it will search for a file where it says `\begin{document}`

At the Moment the packages does not provide the service yet. But I think I can implement that service

**Comment by torik42** [comment]
> > I don’t know, how exactly the Atom-Latex package works. According to the README it also allows to use the magic command to specify the root file.
> 
> Just go and read on. If the manual fails it will search for the magic command. If this fails as well, if will look for a `.latexconfig` if this fails it will search for a file where it says `\begin{document}`

Yes, I had seen that. I just wanted to say that this would also solve my use case.

Now I tested the Atom-Latex package. And I really like the syntax highlighting and auto-completion. But it seems that it only allows one root file per window (or per project?). Thus, you cannot have two distinct projects open and use it (or linter-yalafi if we use their mechanism) at the same time. Is that right? Instead, if you want to lint another file, you would use their `build-here` command (which also compiles the document) and then save the document manually to call the linter. This seems bad to me.

> At the Moment the packages does not provide the service yet. But I think I can implement that service

In case you do that, I would suggest adding an option to the linter-yalafi preferences to choose whether to use their service or not. This would be great for users of Atom-Latex I think.
